src/js/components/projects/projectForm.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
I'm not sure why this is needed? Or if it is, it'd be nice to only use it for the relevant code section.

Reply:
It's there since the api is requiring params in camelCase to create a project (I'm guessing temporarily) such as the `commit_message` and `release_notes` as seen here: https://github.com/SFDO-Tooling/MetaShare/pull/69/commits/9d37af049d0e7857e1efa7fc60707824e0a5c7dd#diff-177bdc19a36cd6b157d923a44b273f69R32